### ISSUE TITLE
ROS2 driver pointcloud data lost

### ISSUE THREAD

**Issue by ahuazuipiaoliang** [ouster-lidar/ouster-ros#190]
**Describe the bug**
sometimes the pointcloud data lost when use the ROS2 driver for OS1 128 REV7

**To Reproduce**
Steps to reproduce the behavior (steps below are just an example):
1. source ros environment
2. compile the project workspace
3. source the project workspace
4. ros launch ouster_ros sensor.launch.xml
5. automatically open Rviz2
6. observe the issue ...

**Screenshots**
![image](https://github.com/ouster-lidar/ouster-ros/assets/35034146/c9d7e961-6287-4e0a-877f-6671337b6ed9)
abnormal data
![image](https://github.com/ouster-lidar/ouster-ros/assets/35034146/50bf4d19-694e-4e91-8b66-86f6b332bd22)
normal data

**Platform (please complete the following information):**
- Ouster Sensor? 
OS1 128 REV7
- Ouster Firmware Version? 
![image](https://github.com/ouster-lidar/ouster-ros/assets/35034146/662aa504-22e4-4454-ba04-77125244e920)
- ROS version/distro? 
Humble
- Operating System?
Ubuntu 22.04
- Machine Architecture? 
x86
- git commit hash (if not the latest).
13171a84b986cae489f644e86fb4a57a4b373cf9

**Comment by ahuazuipiaoliang** [comment]
more abnormal screenshots
![image](https://github.com/ouster-lidar/ouster-ros/assets/35034146/2d490618-a518-4a4c-adfe-43fde8fd6087)
![image](https://github.com/ouster-lidar/ouster-ros/assets/35034146/3ae0305d-d8a6-4dc2-9563-5e98eee22165)
![image](https://github.com/ouster-lidar/ouster-ros/assets/35034146/b20e5978-d624-47d9-8758-4c3ae7487b69)

**Comment by Samahu** [comment]
Hi @ahuazuipiaoliang, we do have have some upcoming changes the would help with the issue. In the meantime, could you try to increase the receive buffer  on our system and see if that helps out:
```bash
sudo sysctl -w net.core.rmem_default=16777216
sudo sysctl -w net.core.rmem_max=16777216
```

**Comment by ahuazuipiaoliang** [comment]
@Samahu I found the data lost problem was caused by the hardware, since I switched to a new LiDAR the problem disappeared. I will close the issue.

**Comment by Samahu** [comment]
Fixed per #262
